### PR TITLE
Update documentation regarding modern and classic search experiences

### PR DESCRIPTION
**Pull request by wobba**
Updated existing documentation to align with correct behavior.
Added new documentation related to deprecation of using a custom default result source in modern search where vertical admin is available.

### CODE REVIEW

File: SharePoint/SharePointOnline/differences-classic-modern-search.md [hunk 1]
```diff
@@ -40,7 +40,8 @@ You use the <a href="https://go.microsoft.com/fwlink/?linkid=2185219" target="_b
   - Refinable
   - Company name extraction (to be deprecated as of November 15th, 2019)
 
-- The modern search experience only shows results from the default result source. If you change the default [result source](manage-result-sources.md), both search experiences are impacted.
+- In environments where vertical configuration is available the modern search experience only shows results from the standard result source (Local SharePoint Results). To learn more, see [Manage search verticals](/microsoftsearch/manage-verticals).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/differences-classic-modern-search.md:43]:
```suggestion
- In environments where vertical configuration is available the modern search experience only shows results from the standard result source (local SharePoint results). To learn more, see [Manage search verticals](/microsoftsearch/manage-verticals).
```

File: SharePoint/SharePointOnline/manage-query-rules.md [hunk 2]
```diff
@@ -34,10 +34,10 @@ You can create query rules at different levels: for the whole tenant, for a site
 
 You can configure query rules for one or more result sources, and you can specify a time period for when the query rule is active.
 
-SharePoint has both a classic and a modern search experience. Only query rules that return promoted results and that are defined for the default result source can affect the modern search experience. Query rules which do query re-writes are not supported for modern experiences. Users might see such promoted results on the **All** tab on the search results page when they search across all of SharePoint. [Learn more about the differences between the classic and modern search experiences in SharePoint](differences-classic-modern-search.md).
+SharePoint has both a classic and a modern search experience. Only query rules that return promoted results can affect the modern search experience. Query rules which do query re-writes are not supported for modern experiences. Users might see such promoted results on the **All** tab on the search results page when they search across all of SharePoint. [Learn more about the differences between the classic and modern search experiences in SharePoint](differences-classic-modern-search.md).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/manage-query-rules.md:37]:
```suggestion
SharePoint has both a classic and a modern search experience. Only query rules that return promoted results can affect the modern search experience. Query rules which do query rewrites are not supported for modern experiences. Users might see such promoted results on the **All** tab on the search results page when they search across all of SharePoint. [Learn more about the differences between the classic and modern search experiences in SharePoint](differences-classic-modern-search.md).
```

File: SharePoint/SharePointOnline/manage-query-rules.md [hunk 3]
```diff
@@ -34,10 +34,10 @@ You can create query rules at different levels: for the whole tenant, for a site
...
   
 > [!IMPORTANT]
-> In December 2021, query rules with promoted results are deprecated for organization scoped queries in favor of [Bookmark and Q&A answers](/microsoftsearch/manage-bookmarks) in modern search experiences. Examples of organization scoped queries are SharePoint Home and office.com. <p>Promoted results will continue to work for all classic search experiences and for sites and hub sites with modern search experiences until Bookmark answers are available for these scenarios. Promoted results will also continue to work for environments where Bookmark and Q&A answers are not available.
+> In December 2021, query rules with promoted results are deprecated for organization scoped queries in favor of [Bookmark and Q&A answers](/microsoftsearch/manage-bookmarks) in modern search experiences. Examples of organization scoped queries are SharePoint Home and office.com. <p>Promoted results will continue to work for all classic search experiences and for sites and hub sites with modern search experiences. Promoted results will also continue to work for environments where Bookmark and Q&A answers are not available.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/manage-query-rules.md:40]:
```suggestion
> In December 2021, query rules with promoted results were deprecated for organization-scoped queries in favor of [Bookmark and Q&A answers](/microsoftsearch/manage-bookmarks) in modern search experiences. Examples of organization-scoped queries are SharePoint Home and Office.com. <p>Promoted results will continue to work for all classic search experiences and for sites and hub sites with modern search experiences. Promoted results will also continue to work for environments where Bookmark and Q&A answers are not available.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @wobba, kindly commit the minor unrelated changes. Many thanks.

File: SharePoint/SharePointOnline/differences-classic-modern-search.md (hunk 1 above)
Comment (wobba) [reply SharePoint/SharePointOnline/differences-classic-modern-search.md:43]: The source is named "Local SharePoint Results" in all UI's, hence the capital letters per word.

Comment (JohanFreelancer9) [reply SharePoint/SharePointOnline/differences-classic-modern-search.md:43]: @wobba, thank you for the explanation. Let's leave it as is. Could you please mark the suggestion as "Resolved"? Thanks again.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by wobba** [comment]
@SerdarSoysal @scanum What's the progress of merging this as I need it updated before a message center post goes out.

**Comment by scanum** [comment]
@tiburd Hi Tina! Hope all is well. Is this something you can help us with? Thank you!

**Comment by tiburd** [comment]
@wobba 
Hi, Mikael,

As a Microsoft-internal submitter, we recommend you submit changes via private repositories, as described in [the authoring guide](https://review.docs.microsoft.com/en-us/Office-Authoring-Guide/office-docs-authoring-overview?branch=main).

We're closing this pull request (PR). 

Please resubmit your proposed changes via the SharePoint private repo, and note that your branch should merge with main instead of public.

Thank you!

cc: @scanum

**Comment by tiburd** [comment]
@wobba 
Thank you - sincere apologies for the delay in providing the feedback.

**Pull request closed without merging**